### ISSUE TITLE
c samples - mingw64 on windows - had to edit Makefile to run samples

### ISSUE THREAD

**Issue by jojomickymack** [megamarc/Tilengine#20]
thank you for the cool project, I can't wait to play with this! Wanting to play with the c samples, I ran the windows install.bat which correctly copied the 64bit dlls into the samples directory, but then ran into some issues with the make file.

on lines 8 and 9
```
    CFLAGS  = -m32 -msse2 -std=c99 -O2 -I"../../include"
    LDFLAGS = ../../lib/win32/Tilengine.lib -s  
```

I had to remove the -m32 flag and change win32 to x64. I was then able to compile and run all the cool samples.

It occurred to me that not everyone would know to do this, and it's far more likely someone will be compiling on 64bit windows I think. Maybe the Makefile should target 64bit windows?

**Comment by megamarc** [comment]
Thanks for your comment! At this moment, the makefile for windows only supports mingw32 for 32-bit architecture (Microsoft Visual C++ supports both). mingw64 is not the same environment and not trivial to set up, so I don't have experience with it. Given that you have experience targeting mingw32 and mingw64, could you create a pull request with the modified windows makefile?

**Comment by jojomickymack** [comment]
I made the modification in my pull request. After comparing the contents of the window32 and window64 downloads, I realized that the 64bit one doesn't even have a lib/win32 directory - just lib/64. The makefile in the 64bit download should have those 2 lines changed in my pull request, the 32bit should remain as it is in master.
If someone intends to use a 32bit compiler on a 64bit machine, they'll certainly choose the 32bit download and everything will work (accept for the install.bat file). Still, the correct dlls will be included with the download.
